src/main/java/at/ac/tuwien/kr/alpha/Util.java

```diff
@@ -33,4 +58,16 @@
 			}
 		}
 	}
+
+	/**
+	 * Constructs a {@link Set} containing only the given element.
+	 * 
+	 * @param element
+	 * @return
+	 */
+	public static <E> Set<E> set(E element) {
```

Comment:
There's [`Collections.singleton`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singleton-T-) for that.